<!-- Launches shelf cropper. -->
<launch>
  <arg name="debug" default="false" />
  <param name="/use_sim_time" value="false"/>
  <node ns="perception" pkg="pr2_pick_perception" type="shelf_cropping_service_node" name="shelf_cropper" respawn="true" output="screen"> 
   <param name="Debug" value="$(arg debug)" />
   <param name="Width1" value="0.2671" />
   <param name="Width2" value="0.2991" />
   <param name="Height1" value="0.26" />
   <param name="Height2" value="0.23" />
   <param name="Depth" value="0.43" />
   <param name="bottom_crop_offset" value="0.055" />
   <param name="top_crop_offset" value="0.04" />
   <param name="left_crop_offset" value="0.025" />
   <param name="right_crop_offset" value="0.025" />
   <param name="depth_far_crop_offset" value="0.08" />
   <param name="depth_close_crop_offset" value="0.02" />

   <remap from="/pc_topic" to="/head_mount_kinect/depth_registered/points"/>
  </node>
</launch>
